frameworks/hdfs/tests/config.py

```diff
@@ -20,58 +22,48 @@
 DOCKER_IMAGE_NAME = "mesosphere/hdfs-client:2.6.4"
 KERBERIZED_CLIENT_IMAGE_NAME = "nvaziri/kerberized-hdfs-client:dev"
 KERBERIZED_CLIENT_MEM_SIZE = 512
+KEYTAB = "hdfs.keytab"
```

Comment:
Let's just use the value as specified in the `json` file.

Reply:
The only this is here is because it helps avoid parsing the path of the keytab when a client wants to do `kinit` (https://github.com/mesosphere/dcos-commons/blob/auth_test/frameworks/hdfs/tests/test_auth.py#L107)